core/driver/sources/src/main/scala/org/virtuslab/ideprobe/wait/ExecutionBackoff.scala

```diff
@@ -0,0 +1,23 @@
+package org.virtuslab.ideprobe.wait
+
+import scala.concurrent.duration.FiniteDuration
+
+class ExecutionBackoff(maxFrequency: FiniteDuration) {
```

Comment:
Personally, I would rename it to something like `Throttle`

Reply:
done